.env

```diff
@@ -5,3 +5,5 @@ REACT_APP_AWS_API_ACCESS_KEY="AKIAYJJWW6AQ47ODATHN"
 REACT_APP_AWS_API_ACCESS_SECRET="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"
 REACT_APP_AWS_X_API_KEY="z9dReS5UtHu7iTrUsTuWRozLthi3AxOZlvobrIdr14"
 REACT_APP_AWS_API_ENDPOINT="https://beta.api.uniswap.org/v1/graphql"
+REACT_APP_GENIE_API_URL="https://genie-production-api.herokuapp.com"
+REACT_APP_GENIE_V3_API_URL="https://v3-rewrite-staging.herokuapp.com"
```

Comment:
both are needed for making calls in the app and both of these apis are publicly useable